app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,80 @@
+// TODO
+// 1. Read Collection - GET /tasks => 완료
+// 2. Read Item - GET /tasks/{id} => 완료
+// 3. Create - POST /tasks => 완료
+// 4. Update - PUT/PATCH /tasks/{id} => 완료
+// 5. Delete - DELETE /tasks/{id} => 완료
+
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * 할 일 Controller
+ * 할 일에 대한 생성/수정/삭제 기능 제공
+ */
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+
+    private HashMap<Long, Task> tasks = new HashMap<>();
+    private long newId = 0L;
+
+    @GetMapping
+    public ArrayList<Task> list() {
+        return new ArrayList<>(tasks.values());
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> get(@PathVariable("id") Long id) {
+        Task task = tasks.get(id);
+        if(task != null) {
+            return new ResponseEntity<>(task, HttpStatus.OK);
+        } else {
+            return new ResponseEntity<>(null, HttpStatus.NOT_FOUND);
+        }
```

Comment:
```suggestion
        tasks.get(id)
            .getOrElse(new ResponseEntity<>(null, HttpStatus.NOT_FOUND));
```

이렇게 작성할 수 있나요? 이것보다는 예외로 처리하는게 더 좋을 것 같은데 그것은 과제풀이 영상을 참고하시면 좋을 것 같습니다